### ISSUE TITLE
Stop Training When Validation Loss Increases

### ISSUE THREAD

**Issue by sasank-desaraju** [BRIO-lab/LitJTML#9]
Implement stopping training when val loss increases. I think we can do this through Callbacks. Maybe create a new callback class in the callbacks.py file so we can toggle whether to use this feature or not? Or just put it in the sole extant class bc "we will always use this".
